lib/models/artist.dart

```diff
@@ -11,6 +11,7 @@ class Artist {
   bool? _isMemeVerified;
   bool? _isVerified;
   String? _name;
+  List<String> _alternateNames = [];
```

Comment:
could you please add these info:
```dart
_facebookName = artistInfo['facebook_name'];
_instagramName = artistInfo['instagram_name'];
_twitterName = artistInfo['twitter_name'];
_description = artistInfo['description']['plain'];
```

Reply:
i  will add this info ,but i was thinking in something differente like a object called socialNetwork , i thinks it's better.

Reply:
i will do that in other PR , to keep things simple.